cmd/otelcontribcol/go.mod

```diff
@@ -426,9 +426,9 @@ require (
 	github.com/dennwc/varint v1.0.0 // indirect
 	github.com/devigned/tab v0.1.1 // indirect
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
-	github.com/digitalocean/godo v1.109.0 // indirect
+	github.com/digitalocean/godo v1.113.0 // indirect
 	github.com/distribution/reference v0.5.0 // indirect
-	github.com/docker/docker v25.0.5+incompatible // indirect
+	github.com/docker/docker v26.0.1+incompatible // indirect
```

Comment:
I think this breaks the build? Since failed build lint check show the otelcontribcol is not built successfully.

Reply:
The failed build is due to `github.com/google/cadvisor`

```
# github.com/google/cadvisor/container/docker/utils
../../../../../pkg/mod/github.com/google/cadvisor@v0.49.0/container/docker/utils/docker.go:48:42: undefined: dockertypes.Info
../../../../../pkg/mod/github.com/google/cadvisor@v0.49.0/container/docker/utils/docker.go:57:44: undefined: dockertypes.Info
../../../../../pkg/mod/github.com/google/cadvisor@v0.49.0/container/docker/utils/docker.go:77:43: undefined: dockertypes.Info
../../../../../pkg/mod/github.com/google/cadvisor@v0.49.0/container/docker/utils/docker.go:86:48: undefined: dockertypes.ImageSummary
```

I try the latest version of cadvisor 0.49.1 again, it stills failed in same reason.

Should this be solved by cadvisor first, then continue this pr? @songy23

Reply:
Yes. Problem is cadvisor depends on `github.com/docker/docker v24.0.9+incompatible` https://github.com/google/cadvisor/blob/master/go.mod#L14 and that version does not work with the one here. Until cadvisor adopts `v26.0.1+incompatible` this PR cannot move forward.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33170

Reply:
Is there a reason they stick on the old version or can we bump it in their project?